rules/transition_support.bzl

```
@@ -74,6 +102,19 @@ def _min_os_version_or_none(attr, attr_platforms, platform, attr_platform_type):
 
     return None
 
+def _objc_library_transition_values(settings, platform_type):
```

nit: im not sure the best name here but since this is related to the transition of apple platform moving to rules vs. defined in Bazel maybe something like this is more clear:

```suggestion
def _apple_platforms_base_values(settings, platform_type):
```

Alternatively we could try to match: https://github.com/bazelbuild/rules_apple/blob/master/apple/internal/transition_support.bzl#L440-L455